cmd/bnsd/app/codec.proto

```diff
@@ -30,6 +30,8 @@ message Tx {
   cash.FeeInfo fees = 1;
   repeated sigs.StdSignature signatures = 2;
   // ID of a multisig contract.
+  // nil array results in multisig being ignored, basically the same behaviour
```

Comment:
Is "nil array" the same as "empty array"? I think the later would be easier to understand outside of go world. Thanks!

Reply:
done